lib/modules/platform/gerrit/client.ts

```diff
@@ -63,9 +63,13 @@ class GerritClient {
     refreshCache?: boolean,
   ): Promise<GerritChange[]> {
     const opts: HttpOptions = {};
-    if (!refreshCache) {
+    /* v8 ignore start: temporary code */
+    if (refreshCache) {
```

Comment:
I suggest bringing this comment from #35056, to address the confusion around this option name.

```suggestion
    // TODO: should refresh the cache rather than just ignore it
    if (refreshCache) {
```

Reply:
The comment applies to bitbucket too, btw.

Reply:
@zharinov please update this section to match what you did for Gerrit:

https://github.com/renovatebot/renovate/blob/f2fffbf8f5823434f2112bdf7dc2c02c18108b33/lib/modules/platform/bitbucket-server/index.ts#L343-L349

The code was exactly the same, would be nice if they were kept the same.